test/unit/sdam/server_discovery_and_monitoring.spec.test.js

```diff
@@ -19,7 +19,7 @@ const { eachAsyncSeries, ns } = require('../../../src/utils');
 
 const chai = require('chai');
 chai.use(require('chai-subset'));
-chai.use(require('../../functional/spec-runner/matcher').default);
+chai.use(require('../../tools/spec-runner/matcher').default);
```

Comment:
Should we use the shared chai-addon.js file here? Actually we should consolidate any `chai.use` calls, I see 20 instances, quick win?

Reply:
sure, good call; I assume you're just suggesting "require(</path/to/chai-addons.js>)?

Reply:
yep!

Reply:
turns out this is already in the mocharc file, so no need for those lines at all